### PR TITLE
RARO-345: subject areas

### PR DESCRIPTION
**Pull request by msingh05**

### CODE REVIEW

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_article_page.scss [hunk 1]
```diff
@@ -158,6 +158,37 @@ main article {
         }
       }
 
+      .subjects {
+        > div {
+          width: 100%;
+
+          // flex box layout can be left-right with wrapping, cannot do top-down columns
+          -webkit-column-count: 2;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_article_page.scss]: I think there is a kube way to do this. Avoids all the webkit, moz prefixes

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
   &:after {
     position: relative;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: position:relative is here twice

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
     content: "+";
-    top: 12px;
+    top: 4px;
     width: 20px;
     height: 20px;
     font-size: 16px;
```
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: use the standard variable for font-size.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss (hunk 2 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: I don't see used it twice.

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss (hunk 3 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_component.scss]: done

File: src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_article_page.scss (hunk 1 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/_article_page.scss]: if you can find example please let me know.

**Pull request merged**